### PR TITLE
New release v1.2.1 (added draft EIP-1108 alt_bn128 gas costs)

### PR DESCRIPTION
**Pull request by holgerd77**
After reading [this tweet](https://twitter.com/nanexcool/status/1133808384204050433) I thought it would actually be fun to directly start integrating a first EIP in the `v4` release of the VM.

There is all this talk on the [AllCoreDevs](https://gitter.im/ethereum/AllCoreDevs) Gitter channel on reference implementations of EIPs, since this time there seems to be the need of some "Champion", so 1-2 implementations of EIPs until they get accepted, as far as I understand also for the more simple onces (since it wouldn't make sense to draw some line here (where to draw?).).

`EIP-1108` is a good fit for a start, since while really simple it has large benefits and would help Aztec privacy protocol a lot, on the same time it's really easy for us here to integrate.

I think this also has very much some real world use, since people then have some first tool to run there contracts against which can be used to develop some feeling for the changing gas costs and investigate further respectively prepare.

Beside it would draw some attention to the VM 😋 (I think it would really deserve some more). Our VM is actually generally very much suited for experimentation like this, since JavaScript is pretty accessible for many people. Maybe we can establish a bit in the whole process as some research VM and eventually also get some implementations for free along the process. 😄 😛 Especially for people coming from the outside with not too deep VM knowledge yet it's then pretty valuable to have a first EIP implementation done which one can reference and go along on own implementations.

I've also directly integrated the release bump and preparation, since I think these one-by-one additions of EIP parameters for `Istanbul` on this library will be pretty deterministic in release planning and should ease the process a bit.

### CODE REVIEW

**Comment by s1na** [comment]
Great initiative Holger!

Yeah as far as I know champions should provide a reference implementation of their proposals. My guess is that the client of choice is usually one of [aleth, geth, parity]. Not sure if JS is a good target for reference implementations, but I agree it's great for experimentation, and this is something we can definitely help with.

As for EIP-1108, there will be an [AMA](https://en.ethereum.wiki/roadmap/istanbul#eip-am-as-on-all-core-devs-gitter) for the elliptic curve cluster today, it'd be great if we could see how it fares, as there are a couple of more proposals in that cluster.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.2.1] - 2019-06-03
+
+- Added `Istanbul` HF candidate [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108)
+  (`DRAFT`) updated `alt_bn128` precompile gas costs (see `hardforks/istanbul.json`)
```
Comment (s1na) [inline CHANGELOG.md]: `Updated ...` can be a separate item

Comment (holgerd77) [reply CHANGELOG.md]: That was actually meant as one sentence "added the updated precompile gas costs being a Istanbul HF candidate", can we go with the current version eventually? 😎🙂

Comment (s1na) [reply CHANGELOG.md]: Ah misread that

**Comment by holgerd77** [comment]
Ok, just released [v1.2.1](https://github.com/ethereumjs/ethereumjs-common/releases/tag/v1.2.1).

**Pull request merged**